canonical-websites/discourse-docs
# Use standard Talisker & run settings, & cleanup

- Use the Talisker settings for development and production that we now use in all the other Python apps.
- Upgrade run script to latest version.
- Remove unneeded code from the app

QA
--

``` bash
./run
```

Check you see "= Running talisker with development settings =" and that the site runs on http://localhost:8029.
